Sources/AccioKit/Services/XcodeProjectIntegrationService.swift

```diff
@@ -241,4 +242,19 @@ final class XcodeProjectIntegrationService {
 
         try projectFile.write(path: Path(xcodeProjectPath), override: true)
     }
+
+    private func verifyBundleVersion(of product: FrameworkProduct) throws {
+        let plistURL = product.frameworkDirUrl.appendingPathComponent("Info.plist")
+        let data = try Data(contentsOf: plistURL)
```

Comment:
If the Info.plist file doesn't exist at all, as is the case for [SwiftEntryKit](https://github.com/huri000/SwiftEntryKit/tree/master/Source), we should create it.

Does this initializer `throw` or does it return empty `Data` if the file is missing? If the latter is the case, one would need to first create the file, right?

Reply:
No the Info.plist file will always exist in the final `MyDependency.framework` folder, only the key `CFBundleVersion` is missing if the project it self is build without having a `Info.plist` file.